internal/daemonset/daemonset.go

```diff
@@ -204,6 +204,11 @@ func (dc *daemonSetGenerator) SetDriverContainerAsDesired(ctx context.Context, d
 		container.VolumeMounts = append(container.VolumeMounts, firmwareVolumeMount)
 	}
 
+	serviceAccountName := mod.Spec.ModuleLoader.ServiceAccountName
```

Comment:
Wondering if the SA management also belongs upstream - without the SCC. WDYT?

Reply:
This is a very good point, I do think that we should bring the SA management upstream as well. 

> It is a good practice to have a SA  per workload and we shouldn't rely on the default SA (when the KMM user don't define any SA). The (admin) users can change the default SA to fulfill other needs for different types of workloads at any point.